src/lib/logger.coffee

```diff
@@ -52,17 +53,25 @@ exports.log = ->
 	publish(arguments...)
 
 do ->
+	_lock = new Lock()
```

Comment:
Yeah, we have a bunch of rwlocks in several parts of the supervisor.
The one in logger was for a different purpose and is no longer necessary. I think it's fine to have this here.